File: week-02/day-01/draw_chess_table.py
# Crate a program that draws a chess table like this
#
# % % % % 
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % % 
#  % % % %
#
for i in range(8):
    if i % 2 == 0:
        a = "%"
        b = " "
    else:
        a = " "
        b = "%"
    for j in range(4):
        print(a+b, end="")
    print()

File: week-07/day-1/draw-chess-table.js
// Crate a program that draws a chess table like this
//
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % % 
//  % % % %
//

'use strict';

for(let i = 0; i < 8; i++){
    let line = '';
    for(let j = 0; j < 8; j++){
        if((i + j) % 2 === 0){
            line += '%';
        } else {
            line += ' ';
        }
    }
    console.log(line);
}